explorer/templates/explorer/preview_pane.html

```diff
@@ -27,22 +27,22 @@
                                 {% if data %}
                                     <a title="Show row numbers" id="counter-toggle" href="#"><i class="bi-hash"></i></a>&nbsp
                                 {% endif %}
-                                {% blocktrans trimmed with duration=duration|floatformat:2 %}
+                                {% blocktranslate trimmed with duration=duration|floatformat:2 %}
                                     Execution time: {{ duration }} ms
-                                {% endblocktrans %}
+                                {% endblocktranslate %}
                             </div>
                             <div class="col text-end">
                                 <span class="me-1">
                                     {% if rows > total_rows %}
-                                        {% trans "Showing" %}&nbsp;
+                                        {% translate "Showing" %}&nbsp;
                                         <input class="rows-input" type="text" name="rows" id="rows"
                                                value="{{ total_rows }}"/>
                                     {% else %}
-                                        {% trans "First" %}&nbsp;
+                                        {% translate "First" %}&nbsp;
                                         <input class="rows-input" type="text" name="rows" id="rows"
                                                value="{{ rows }}"/>
                                     {% endif %}
-                                    {% blocktrans %}of {{ total_rows }} total rows.{% endblocktrans %}
+                                    {% blocktranslate %}of {{ total_rows }} total rows.{% endblocktranslate %}
```

Comment:
Not sure if this works or if the variable needs to be redefined in the context of the block tag.

Reply:
@marksweb what do you mean? If it works, it works, right?

Reply:
@chrisclark I've not ran it, so couldn't confirm.

Reply:
Works great. 
<img width="237" alt="image" src="https://github.com/explorerhq/django-sql-explorer/assets/693700/48756175-6ca4-4374-b44e-44914ad15c7d">